openprescribing/media/js/src/analyse-chart.js

```diff
@@ -543,10 +544,7 @@ var analyseChart = {
     for (var i = 0; i < data.length; i++) {
       entry = data[i];
       // We include an org both if its ID is selected, or if it belongs to a
-      // CCG which is itself selected.  This logic will need extending if we
-      // add other org types to the Analyse form, so we deliberately throw an
-      // 'Unhandled orgType' error elsewhere if we're given an org type we
-      // don't recognise.
+      // CCG which is itself selected
```

Comment:
s/CCG/grouping (or org)/

Reply:
No, this is specific to the practice/CCG relationship and doesn't apply more generally